docs/guides/implementation-guides/how-to-implement-dependency-injection.md

```diff
@@ -7,9 +7,120 @@ import MvvmArchitectureDiagram from '/img/gitbook-import/assets/image (3) (1) (2
 
 # How To Implement Dependency Injection
 
-Content in preparation.
+:::info
+For views you can make a simple [ViewLocator](../../concepts/view-locator) which will automatically resolve the view from the Locator.
```

Comment:
Please don't mention this unless you are going to show how to do this. It's somewhat extraneous information and will likely just frustrate/confuse people. Further more since most people are care about VM injection, this can probably go at the end. 

But adding some short sample code with a custom view locator should be fairly straight forwards.

Reply:
The link redirect to a documentation page with sample code already. 

I see your point with frustrating/confusing people, so I'll remove this part.

Reply:
The link it points to contains nothing about using a view locator with with DI. It's just pure reflection. Which is why I said that. This page is purely about using DI with Avalonia.